usmqe_tests/conftest.py

```diff
@@ -320,6 +321,32 @@ def fill_cpu():
     return measure_operation(fill_cpu)
 
 
+@pytest.fixture(scope="session")
+def application():
+    app = Application(
+        hostname=CONF.config["usmqe"]["web_url"].split('/')[-1],
+        scheme="http",
```

Comment:
Here I would use `urlparse` from `urllib` like in this example:

```
In [2]: from urllib.parse import urlparse

In [3]: url = urlparse('https://mbukatov-usm2-server.usmqe.lab.eng.brq.redhat.com')

In [4]: url.scheme
Out[4]: 'https'

In [5]: url.hostname
Out[5]: 'mbukatov-usm2-server.usmqe.lab.eng.brq.redhat.com'
```

so that you can set both `hostname` and `scheme` here.

Btw how did it work when the scheme is hardcoded?